src/test/java/com/google/api/codegen/testsrc/libraryproto/library.proto

```diff
@@ -499,6 +499,9 @@ message Book {
   // Tests Python doc generation: should generate a dummy file for shared_type
   // resource, but *not* its import, other_shared_type
   google.test.shared.data.Used resource = 29;
+
+  // For testing accessing map fields in samplegen
+  map<bool, string>  map_bool_key = 30;
```

Comment:
No need to do this now, or maybe at all, but it sure would be nice if we had consistent and informative naming for these maps: wg `map_bool_string`, `map_string_message`, `map_int_string`, etc.....

Reply:
Hmmmm, good point :)